### DISCUSSION TITLE
Should OBO Foundry pursue non-profit status?

### DISCUSSION THREAD

**Discussion by matentzn** [OBOFoundry/OBOFoundry.github.io#2477] [General]
For 20 years, the OBO Foundry has been run as a volunteer organisation. Occasionally, core infrastructure has been funded by grants awarded to OBO Operations Committee members, but the majority of the work has been done on a volunteering basis. This means, on the one side, that the OBO community could organise itself from the bottom up, resulting in a network of passionate scientists and developers that contribute their resources to a common cause. On the other hand, the lack of bespoke resources and the reliance on scarce volunteer hours sometimes makes it very slow to progress. For this reason, a debate has started on whether it may be worth the effort to form a non-profit, which is able to receive funds (donations) which can be spent on core infrastructure. The Governance team, driven by members of our community including @sabrinatoro, @handemcginty and @mellybelly have laid the groundwork for organisational change that would be necessary for such a move. For now, though, we still do not have a good feel _ whether the effort is worth it to reach our goal_ (core funding for core OBO infrastructure and processes), or whether there are more efficient means to get these resources. What is your opinion?

**Comment by hoganwr** [comment]
This is an important discussion so thank you for getting it going. I'd be
interested to know what other means there are--besides status quo and
nonprofit status--to assess whether they're more or less efficient.  I'm
also curious to know what sources of funds the non-profit would have other
than the donations mentioned.  And who would be donating and from
what funds?  Is one idea that we'd have subawards to the non-profit on our
grants?

**Comment by nlharris** [comment]
As I've expressed before, it's not a question of should we or shouldn't we pursue becoming a non-profit; the question is, who would lead/do the significant work required to accomplish this (without getting paid for that work)? (Not me, that's for sure!)

**Reply by matentzn** [reply]
Yeah, it's a very important question. Is there a way to secure a more constant stream of funding to establish a more mature, faster moving organisation, to avoid a full blown non-profit effort?

**Comment by hoganwr** [comment]
Nico, thanks for confirming that I've understood the question / key issue
correctly.  Nomi, I agree: I also understood the status quo as the least
preferred option.

I look forward to hearing folks' thoughts on options in between status quo
and full-blown non-profit, and their relative advantages and
disadvantages.  I'll be sure to share any ideas of my own (at present, none
sufficiently formulated, admittedly).

**Comment by hoganwr** [comment]
I'm bumping this discussion because there was significant confusion on the OFOC call last week.  We do not need a new workgroup to study the issue.  We have the report of an existing governance group, which was presented by Hande to OFOC on the 9/19 call (as recorded in the minutes). I am sorry I could not make that call.  Reviewing the minutes, however, the key question is not non-profit status, so that is one source of confusion to clear up.

The key question is whether we form a governance group.  And the rationale for it is in the document.  I reproduce the link here for your convenience: https://docs.google.com/document/d/10dXXYr-Wmu1JK9UfvoKp-Btb29KqUy2WFGHlAvUbrNI/edit

So we should not start yawg (yet another working group) until we have properly concluded the work of the existing one, IMHO.

**Comment by mellybelly** [comment]
The governance task team is awaiting feedback from the OBO-operations committee and would be happy to help design or lead a non-profit strategy and/or formalize a governance group. Maturing governance is necessary as part of non-profit creation, and was one of the main rationales for instantiating the governance task team in the first place. We also hosted a community survey and several town halls on this topic.
